style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  min-height: 100vh;
+  background-color: #d3d3d3;
+  font-family: cursive;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.cover {
+  margin: 0 auto;
+  width: 200px;
+  max-width: 800px;
+  min-width: 400px;
+  height: 480px;
+  background-image:;
+  background-color: #78aaeb;
+  border-radius: 10px;
+  box-shadow: 5px 10px 20px;
+  transform-origin: left;
+  animation: slide 5s ease-in-out normal forwards;
+  position: absolute;
+  z-index: 2;
+}
+::-webkit-scrollbar {
+  display: none;
+}
+.paper {
+  position: relative;
+  width: 50%;
+  max-width: 200px;
+  min-width: 400px;
+  height: 480px;
+  margin: 0 auto;
+  background: #fafafa;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.3);
+  border-radius: 10px;
+  /* overflow: hidden; */
+}
+.paper::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: 60px;
+  background: radial-gradient(#575450 6px, transparent 7px) repeat-y;
+  background-size: 30px 30px;
+  border-right: 3px solid red;
+  box-sizing: border-box;
+}
```

Comment:
can you explain, if  *   has box-sizing property then why use it  here again @nehalskdev

Reply:
I tried without it buy then my page lines started merging with the red line

Reply:
because is is used inside pseudo class of ``` before``` and in pseudo classes universal selector's properties does not apply.